src/main/java/qna/domain/DeleteHistories.java

```diff
@@ -0,0 +1,22 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeleteHistories {
+    private final List<DeleteHistory> deleteHistories;
+
+    public DeleteHistories(ContentType question, Long id, User writer) {
+        deleteHistories = new ArrayList<>();
+        deleteHistories.add(new DeleteHistory(question, id, writer));
+    }
+
+    public List<DeleteHistory> getDeleteHistories() {
+        return new ArrayList<>(deleteHistories);
```

Comment:
방어적 복사 좋습니다 
아래처럼 제공해주면 추가적으로 Collection에 대한 변경도 막을 수 있을 것 같아요
```suggestion
        return Collections.unmodifiableList(deleteHistories);
```

Reply:
네네!! 좋습니다!!